docs/topics/rest-http-README.adoc

```diff
@@ -0,0 +1,80 @@
+
+:runtime: ${runtime}
+
+ifeval::["{runtime}" == "Spring Boot"]
+:localRunCMD: mvn spring-boot:run
+:OSORunCMD: mvn clean fabric8:deploy -Popenshift -DskipTests
+:guideURL: http://appdev.openshift.io/docs/spring-boot-runtime.html
+:fileLocation: src/main/resources/static/index.html
+endif::[]
+
+ifeval::["{runtime}" == "Eclipse Vert.x"]
+:localRunCMD: mvn vertx:run
+:OSORunCMD: mvn clean fabric8:deploy -Popenshift -DskipTests
+:guideURL: http://appdev.openshift.io/docs/vertx-runtime.html
+:fileLocation: src/main/resources/webroot/index.html
+endif::[]
+
+ifeval::["{runtime}" == "WildFly Swarm"]
+:localRunCMD: mvn wildfly-swarm:run
+:OSORunCMD: mvn clean fabric8:deploy -Popenshift -DskipTests
+:guideURL: http://appdev.openshift.io/docs/wf-swarm-runtime.html
+:fileLocation: src/main/webapp/index.html
+endif::[]
+
+
+
+= ${mission} - ${runtime} Booster
+
+== Run this Booster Locally
+
+To run this booster on your local host:
+
+[source,bash,options="nowrap",subs="attributes+"]
+----
+$ git clone git@github.com:${loggedUser}/${targetRepository}
+
+$ cd ${targetRepository}
+
+$ {localRunCMD}
+----
+
+== Interact with this Booster Locally
+
+To interact with your booster while its running, use the form at `http://localhost:8080` or the `curl` command:
```

Comment:
its -> it's or it is

Also "while it's running" -> "while it's running locally"